### ISSUE TITLE
desktop-app dont show shadow state

### ISSUE THREAD

**Issue by Eugenio13** [MicrochipTech/aws-iot-insight-on-things-desktop-app#9]
Hi
 I have connected my dm990004 board to aws, and I can see in the activity log when I press a key.
But from app Insight   I can not turn on the LEDs
Also the app does not recognize the thing name
I work with the 2.0.0 versión.
Do you have any idea what's happening?
Thank you very much in advance

**Comment by Eugenio13** [comment]
I have continued investigating and it seems that there is some problem when executing the function readInsight ()
And can not read the configuration data, 
does anyone know what should be the content of the .insight file, located in "users / user / AppData / Roaming / SPB_Data /" ?
Thank  in advance..

**Comment by Eugenio13** [comment]
Hello everyone
Everything works
I have reinstalled the app again
I have erased all data in AWS (certificates, policies, users, etc ...)
I have started from scratch, and this time everything has worked automatically.
I think the app only works if everything goes well at first, and does not know how to react to any adverse event.
 close the issue because everything works correctly.
Thanks for the help
